src/utils/num.ts

```diff
@@ -60,23 +106,48 @@ export function toStorageKey(number: BigNumberish): string {
  * Convert hexadecimal string to decimal string
  * @param hex hex-string
  * @returns format: decimal string
+ * @example
+ * ```typescript
+ * const hexString = "0x2fd23d9182193775423497fc0c472e156c57c69e4089a1967fb288a2d84e914";
+ * const result = hexToDecimalString(hexString);
+ * // result = "32507161997631881240494522159444018041090212371621416251492750949915267618964";
+ * ```
  */
 export function hexToDecimalString(hex: string): string {
   return BigInt(addHexPrefix(hex)).toString(10);
 }
 
 /**
- * Remove hex string leading zero and lowercase it
- * @example '0x01A...' -> '0x1a..'
+ * Remove hex string leading zero and lowercase it;
  * @param hex hex-string
  * @returns format: hex-string
+ * @example
+ * ```typescript
+ * const hexString = '0x01A...'
+ * const result = cleanHex(hexString);
+ * //result = '0x1a..'
+ * ```
  */
 export const cleanHex = (hex: string) => hex.toLowerCase().replace(/^(0x)0+/, '$1');
 
 /**
  * Asserts input is equal to or greater then lowerBound and lower then upperBound.
```

Comment:
Could you replace `then` by `than` (x2)?

Reply:
done change then to than